eip-0010.md

```diff
@@ -0,0 +1,39 @@
+Autolykos v. 2
+===============================
+
+* Author: AsicResearcher (aka BigEvilSoloMiner), Jason Davies, kushti
+* Status: Proposed
+* Created: 03-Aug-2020
+* Last edited: 03-Aug-2020
+* License: CC0
+* Forking: hard fork
+
+# Autolykos 2
+
+Autolykos is Proof-of-Work scheme used in Ergo. It was intended to be memory-hard and pool-restistant. 
+However, it was found that memory-hardness of the scheme can be bypassed.
+
+This document describes Autolykos v. 2, which is improving memory-hardness of the original design.
+
+
+# Changes
+
+Original Autolykos scheme is decribed at [https://docs.ergoplatform.com/ErgoPow.pdf](https://docs.ergoplatform.com/ErgoPow.pdf).
+Proving and verification functions are defined in the Algorithms 1 and 2 of the paper.
+
+The proposed changes are:
+
+Alg. 2, line 4: m || nonce  => pk || w || m || nonce
+Alg. 2, line 5: H(j|M|pk|m|w) => H(j|pk|w|M|m)
```

Comment:
I'm curious, what is the motivation for the change? Why the old version is problematic, and why the new one is the solution?